python/pymasarsqlite/db/addmasarconfigs.py

```diff
@@ -0,0 +1,131 @@
+import os
+import sys
+import sqlite3
+import json
+import ConfigParser
+import argparse
+from pymasarmongo.db import utils
+from pymasarmongo.config._config import masarconfig
+from masarclient import masarClient
+from pymasarsqlite.service.serviceconfig import (saveServiceConfig, retrieveServiceConfigs, retrieveServiceConfigPVs, saveServicePvGroup, retrieveServicePvGroups)
+from pymasarsqlite.pvgroup.pvgroup import (savePvGroup, retrievePvGroups)
+from pymasarsqlite.pvgroup.pv import (saveGroupPvs, retrieveGroupPvs)
+from pymasarsqlite.service.service import (saveService)
+from pymasarmongo.pymasarmongo.pymasar import saveconfig
+from pymasarmongo.pymasarmongo.pymasar import updateconfig
+
+def __loadmasarconfig(config=""):
+    cf = ConfigParser.SafeConfigParser()
+    cf.read([
+        os.path.expanduser('~/.masarservice.conf'),
+        '/etc/masarservice.conf',
+        'masarservice.conf',
+        config,
+        "%s/masarservice.conf" % os.path.abspath(os.path.dirname(__file__))
+    ])
+    return cf
+
+def savePvGroups(json):
+    for group in json:
+        pvgname = group[u'name']
+        pvgdesc = group[u'description']
+        pv_file = group[u'pvlist']
+        pvlist = []
+        if os.path.exists(pv_file):
+            with open(pv_file) as file:
+                for line in file:
+                    pvlist.append(line.strip('\n'))
+        else:
+            print "Pvlist file: " + str(pv_file) + " does not exist."
+            # file not found
+    __sqlitedb__ = os.environ["MASAR_SQLITE_DB"]
+    conn = sqlite3.connect(__sqlitedb__)
+    savePvGroup(conn, pvgname, func=pvgdesc)
+    saveGroupPvs(conn, pvgname, pvlist)
+    conn.commit()
+    conn.close()
+
+def saveSQLiteServiceConfig(json):
+    servicename = "masar"
+    servicedesc = 'machine snapshot, archiving, and retrieve service'
+    __sqlitedb__ = os.environ["MASAR_SQLITE_DB"]
+    conn = sqlite3.connect(__sqlitedb__)
```

Comment:
The 'sqlite3.Connection' object can use the 'with' statement to avoid the for explicit close().

```py
with sqlite3.connect(__sqlitedb__) as conn:
   ...
```